spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -725,6 +519,19 @@ The implementation must produce the _vendor_ `MetricRegistry` when the `Registry
 MetricRegistry vendorRegistry;
 ----
 
+==== Custom Metric Registries
+The implementation must produce the `MetricRegistry` corresponding to the custom-named registry when the `RegistryType` is a custom value. If the custom-named MetricRegistry does not yet exist the implementation must create a `MetricRegistry` with the specified name.
+
+.Example of the application injecting a custom-named registry
+[source, java]
+----
+@Inject
+@RegistryType(type="motorguide")
```

Comment:
The parameter is switched to "scope" in https://github.com/eclipse/microprofile-metrics/pull/667

Reply:
given #667 is changing both type to scope and the MetricRegistry constants, it's probably better to keep "type" here and make both changes in #667 .